static/js/app.js

```diff
@@ -227,11 +211,11 @@ function deleteAliasConfirmation(submitEvent) {
 	});
 }
 
-function submitEmailForwardingPrefsForm(submitEvent) {
-  submitEvent.preventDefault();
+function submitEmailForwardingPrefsForm(changeEvent) {
+  changeEvent.preventDefault();
   const isUserPremium = document.querySelector("body").classList.contains("is-premium");
-  const form = submitEvent.target.closest("form");
-  updateEmailForwardingPrefs(form, submitEvent.target.defaultValue, isUserPremium, submitEvent);
+  const form = changeEvent.target.closest("form");
```

Comment:
I think it's more reliable/performant to obtain a reference to an input's associated form, rather than looking for a form element in the DOM:

```suggestion
  const form = changeEvent.target.form;
```

Reply:
Revised in 02d8f987e4840d0cbe4ee26f5d4b28f353fc555b